.github/workflows/release.yml

```diff
@@ -82,14 +127,23 @@ jobs:
         id: extract_branch
         run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
       - name: Invoke Oclif workflow
-        if: ${{needs.release.outputs.tag-name != ''}}
+        if: ${{needs.release.outputs.tag-name != '' && github.event.inputs.draft-release != 'true'}}
         run: node .github/scripts/trigger-workflow.js
         env:
           WORKFLOW_NAME: 'oclif-release.yml'
           REPO_NAME: twilio/twilio-cli
           REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
           BRANCH_NAME: ${{steps.extract_branch.outputs.branch}}
           INPUTS: '{ "home-brew-branch": "${{github.event.inputs.homebrew-branch}}", "tag-name": "${{needs.release.outputs.tag-name}}", "pre-release": "${{github.event.inputs.homebrew-prerelease}}"}'
+      - name: Invoke Oclif workflow for draft release
+        if: ${{needs.release.outputs.tag-name != '' && github.event.inputs.draft-release == 'true'}}
+        run: node .github/scripts/trigger-workflow.js
+        env:
+          WORKFLOW_NAME: 'oclif-release.yml'
+          REPO_NAME: twilio/twilio-cli
+          REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+          BRANCH_NAME: ${{steps.extract_branch.outputs.branch}}
+          INPUTS: '{ "home-brew-branch": "${{github.event.inputs.homebrew-branch}}", "tag-name": "${{needs.release.outputs.tag-name}}-draft", "pre-release": "${{github.event.inputs.homebrew-prerelease}}"}'
```

Comment:
Are we handling the clean up for homebrew release? because it will also trigger the [commit](https://github.com/twilio/homebrew-brew/commit/b9002c15f226ba79b3336f4b4ff623f84a506407) and release a tag.
How are we handling the `draft-release` tag formula in case of homebrew?

Reply:
Missed that, I'll add cleanup for homebrew too, it'd be along the same lines as in npm. About the tag formula, I saw that `version-draft` fell in the standard category [(here)](https://github.com/Homebrew/brew/blob/master/docs/Node-for-Formula-Authors.md#example), do you think we'd need an update in the formula too?

Reply:
Okay. I think we don't need to update formula but here also draft-release released as a main release. And are we planning to clean s3 assets of draft-release?